extensions.json

```
@@ -744,9 +744,7 @@
     },
     {
       "id": "jeroen-meijer.pubspec-assist",
-      "repository": "https://github.com/jeroen-meijer/pubspec-assist",
-      "version": "2.1.0",
-      "checkout": "v2.1.0"
+      "repository": "https://github.com/jeroen-meijer/pubspec-assist"
```

False positive (`2.1.0` is still the latest official version):

```suggestion
      "repository": "https://github.com/jeroen-meijer/pubspec-assist",
      "version": "2.1.0",
      "checkout": "v2.1.0"
```